### PR TITLE
device-amazon-thor: Modernize APKBUILD

### PR DESCRIPTION
**Pull request by steamp0rt**
I also added a new fb.modes that seems to work

[x] Merge on GitHub (see <https://postmarketos.org/merge>)

### CODE REVIEW

**Comment by steamp0rt** [comment]
Yes

File: aports/device/device-amazon-thor/APKBUILD [hunk 1]
```diff
@@ -1,30 +1,29 @@
...
-makedepends=""
-install="$pkgname.post-install"
+makedepends="devicepkg-dev"
+install=""
 subpackages=""
 source="
 	deviceinfo
-	90-android-touch-dev.rules
 	fb.modes
+	90-android-touch-dev.rules
```
Comment (ollieparanoid) [inline aports/device/device-amazon-thor/APKBUILD]: This file does not get used, `devicepkg-dev` creates the udev rule on its own:
* [`devicepkg_build.sh`](https://github.com/postmarketOS/pmbootstrap/blob/master/aports/main/devicepkg-dev/devicepkg_build.sh)
* [`devicepkg_package.sh`](https://github.com/postmarketOS/pmbootstrap/blob/master/aports/main/devicepkg-dev/devicepkg_package.sh)

Can you remove it from `source=""` and delete the file from the aport?

File: aports/device/device-amazon-thor/fb.modes [hunk 2]
```diff
@@ -1,7 +1,9 @@
-mode "1200x1920-0"
-        # D: 1.116 MHz, H: 0.877 kHz, V: 0.448 Hz
-        geometry 1200 1920 1200 1920 16
-        timings 896149 16 32 25 10 24 2
-        accel false
-        rgba 8/24,8/16,8/8,8/0
-endmode
\ No newline at end of file
+
```
Comment (ollieparanoid) [inline aports/device/device-amazon-thor/fb.modes]: How about removing the empty line here?

**Comment by steamp0rt** [comment]
This works and is ready.

**Comment by steamp0rt** [comment]
Thanks!

**Pull request merged**